### ISSUE TITLE
ng update doesn't check or find @angular/pwd

### ISSUE THREAD

**Issue by egonknapen** [angular/angular-cli#23432]
# 🐞 Bug report

### Command (mark with an `x`)

- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [x] update
- [ ] lint
- [ ] extract-i18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

No idea, it never worked before

### Description

'ng update' shows all packages that could be updated, but @angular/pwd is never in that list, although there is a new version available. As seen below in my environment, there are 4 packages that are still on 14.0.2 and can only be updated manually, not by 'ng update'

## 🔬 Minimal Reproduction

> ng update

## 🌍 Your Environment

<pre><code>
Angular CLI: 14.0.3
Node: 16.13.0
Package Manager: npm 8.3.0
OS: win32 x64

Angular: 14.0.3
... animations, cdk, cli, common, compiler, compiler-cli, core
... forms, platform-browser, platform-browser-dynamic, router
... service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1400.3
@angular-devkit/build-angular   14.0.3
@angular-devkit/core            14.0.2
@angular-devkit/schematics      14.0.2
@angular/pwa                    14.0.2
@schematics/angular             14.0.2
rxjs                            7.5.5
typescript                      4.7.4</code></pre>

**Anything else relevant?**

**Comment by alan-agius4** [comment]
`@angular/pwa` is a schematic that is ran only once. Once the schematic is ran this package is no longer needed as a dependency. Likely this is a left over from an old implementation which did save `@angular/pwa` as a dependency.

Please remove this dependency using:

```
npm uninstall @angular/pwa
```
